libs/langchain/langchain/utilities/tensorflow_datasets.py

```diff
@@ -0,0 +1,108 @@
+import logging
+from typing import Any, Callable, Dict, Iterator, List, Optional
+
+from pydantic import BaseModel, root_validator
+
+from langchain.schema import Document
+
+logger = logging.getLogger(__name__)
+
+
+class TensorflowDatasets(BaseModel):
+    """Access to the TensorFlow Datasets.
+
+    Current implementation can work only with datasets that fit in a memory.
+
+    `TensorFlow Datasets` is a collection of datasets ready to use, with TensorFlow
+    or other Python ML frameworks, such as Jax. All datasets are exposed
+    as `tf.data.Datasets`.
+    To get started see the Guide: https://www.tensorflow.org/datasets/overview and
+    the list of datasets: https://www.tensorflow.org/datasets/catalog/overview#all_datasets
+
+    You have to provide the sample_to_document_function: a function that converts a sample
+     from the dataset-specific format to the Document.
+
+    Args:
+        dataset_name: the name of the dataset to load
+        split_name: the name of the split to load. Defaults to "test".
+        load_max_docs: a limit to the number of loaded documents. Defaults to 100.
+        sample_to_document_function: a function that converts a dataset sample to a Document
+
+    Example:
+        .. code-block:: python
+
+            from langchain.utilities import TensorflowDatasets
+
+            def mlqaen_example_to_document(example: dict) -> Document:
+                return Document(
+                    page_content=decode_to_str(example["context"]),
+                    metadata={
+                        "id": decode_to_str(example["id"]),
+                        "title": decode_to_str(example["title"]),
+                        "question": decode_to_str(example["question"]),
+                        "answer": decode_to_str(example["answers"]["text"][0]),
+                    },
+                )
+
+            tsds_client = TensorflowDatasets(
+                    dataset_name="mlqa/en",
+                    split_name="test",
+                    load_max_docs=MAX_DOCS,
+                    sample_to_document_function=mlqaen_example_to_document,
+                )
+
+    """  # noqa: E501
```

Comment:
Would you mind wrapping the doc-strings and re-enabling the linting rule?

Reply:
@baskaryan  Yeah, about that. It is actually a **big annoying** thing.
The current `black` line size of 88 doesn't make much sense. (do we target the mobile phone developers [by such I mean not the developers that create mobile phone apps, but developers that **develop code** on phones.]?)
I saw computers with such tiny screens only in 90s. I personally set it up to 140 and still have a space.
IMP 120 characters fit everybody. 
Aghhh, just complaining...